src/Analysis/Ast/Impl/Analyzer/PythonAnalyzerSession.cs

```diff
@@ -128,10 +128,6 @@ internal sealed class PythonAnalyzerSession {
             }
 
             var stopWatch = Stopwatch.StartNew();
-            foreach (var affectedEntry in _walker.AffectedValues) {
-                affectedEntry.Invalidate(Version);
```

Comment:
Old code that doesn't required anymore. Also causes the issues invalidating library modules that have been analyzed by previous session.